server/errorstore/errors.go

```diff
@@ -224,12 +229,6 @@ func (h *Handler) handleErrors(ctx context.Context) {
 }
 
 func (h *Handler) storeError(ctx context.Context, err error) {
-	// Skip storing errors due to SCAN issues with Redis (see #3065).
-	// if true here because otherwise we get linting errors for unreachable code.
-	if true {
-		return
-	}
```

Comment:
Note that this PR does reintroduce storing of errors in Redis, if the config allows it.